pandas/core/ops.py

```diff
@@ -440,44 +442,11 @@ def _validate_timedelta(self, name):
                             'of a series/ndarray of type datetime64[ns] '
                             'or a timedelta')
 
-    def _validate_offset(self, name):
-        # assumes self.is_offset_lhs
-
-        if self.is_timedelta_rhs:
-            # 2 timedeltas
-            if name not in ('__div__', '__rdiv__', '__truediv__',
-                            '__rtruediv__', '__add__', '__radd__', '__sub__',
-                            '__rsub__'):
-                raise TypeError("can only operate on a timedeltas for addition"
-                                ", subtraction, and division, but the operator"
-                                " [{name}] was passed".format(name=name))
-
-        elif self.is_datetime_rhs:
-            if name not in ('__add__', '__radd__'):
-                raise TypeError("can only operate on a timedelta/DateOffset "
-                                "and a datetime for addition, but the operator"
-                                " [{name}] was passed".format(name=name))
-
-        else:
-            raise TypeError('cannot operate on a series without a rhs '
-                            'of a series/ndarray of type datetime64[ns] '
-                            'or a timedelta')
-
     def _validate(self, lvalues, rvalues, name):
         if self.is_datetime_lhs:
             return self._validate_datetime(lvalues, rvalues, name)
         elif self.is_timedelta_lhs:
             return self._validate_timedelta(name)
-        elif self.is_offset_lhs:
```

Comment:
why is this offset case not possible? (it may not be tested) but is it possible?

Reply:
Because _TimeOp is only used if `is_datetime_lhs or is_timedelta_lhs`

Reply:
ok, can you doc-string things a bit, will help future readers.